<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>458</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>24 点游戏（Designed by bin）</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhLowercaseOnly</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>310</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> ：</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>370</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
     </font>
    </property>
    <property name="text">
     <string>答案</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_coin">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>金币</string>
    </property>
   </widget>
   <widget class="QLabel" name="num1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/poke_backgrand.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="num2">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>130</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/poke_backgrand.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="num3">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>130</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/poke_backgrand.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="num4">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>130</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/poke_backgrand.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_rooster">
    <property name="geometry">
     <rect>
      <x>728</x>
      <y>28</y>
      <width>72</width>
      <height>72</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>鸡哥</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_set">
    <property name="geometry">
     <rect>
      <x>818</x>
      <y>12</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/setting.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="but_music">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/music_close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="but2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/button_box.png);</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>242</width>
      <height>26</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>363</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/input_box.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label0">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>370</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> ：</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>356</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/button.png);</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_help">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>45</y>
      <width>32</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/photo/help.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_flower">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>44</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>向日葵</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_music">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>32</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
  border: 0px solid #bbb;
}
 
QSlider::sub-page:horizontal {
  background: rgb(255,255,0);
  border-radius: 0px;
  margin-top: 8px;
  margin-bottom: 9px;
}
 
QSlider::add-page:horizontal {
  background: rgb(0,0,0);
  border: 0px solid #777; 
  border-radius: 2px; 
  margin-top: 8px; 
  margin-bottom: 9px; 
}
 
QSlider::handle:horizontal { 
  background: rgb(255,255,255); 
  border: 1px solid rgba(102,102,102,102); 
  width: 8px; 
  height: 8px; 
  border-radius: 5px; 
  margin-top: 2px; 
  margin-bottom: 2px; 
}
 
QSlider::handle:horizontal:hover { 
  background: rgb(255,255,0); 
  border: 1px solid rgba(102,102,102,102); 
  border-radius: 5px; 
}
 
QSlider::sub-page:horizontal:disabled { 
  background: #bbb; 
  border-color: #999; 
}
 
QSlider::add-page:horizontal:disabled { 
  background: #eee; 
  border-color: #999; 
}
 
QSlider::handle:horizontal:disabled { 
  background: #eee; 
  border: 1px solid #aaa; 
  border-radius: 4px; 
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_music_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>12</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>7</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>music info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_music_time">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>26</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_last">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>6</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(170,170,255);
border-image: url(:/photo/music_last.png);
color:white;
border-radius:11px;
border:2px groove gray;
border-style:outset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="but_play">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>6</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/photo/music_pause_white.png);
background-color:rgb(170,170,255);
color:white;
border-radius:11px;
border:2px groove gray;
border-style:outset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="but_next">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>6</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(170,170,255);
border-image: url(:/photo/music_next.png);
color:white;
border-radius:11px;
border:2px groove gray;
border-style:outset;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_file">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>6</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/music_file.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文隶书</family>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/dialog_box.png);</string>
    </property>
    <property name="text">
     <string>     生命不息，奋斗不止！</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>458</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/photo/456.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label4</zorder>
   <zorder>label1</zorder>
   <zorder>label2</zorder>
   <zorder>label3</zorder>
   <zorder>label_coin</zorder>
   <zorder>num1</zorder>
   <zorder>num2</zorder>
   <zorder>num3</zorder>
   <zorder>num4</zorder>
   <zorder>label_rooster</zorder>
   <zorder>but_set</zorder>
   <zorder>but_music</zorder>
   <zorder>but2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label0</zorder>
   <zorder>but1</zorder>
   <zorder>but_help</zorder>
   <zorder>label_flower</zorder>
   <zorder>slider_music</zorder>
   <zorder>label_music_name</zorder>
   <zorder>label_music_time</zorder>
   <zorder>but_last</zorder>
   <zorder>but_play</zorder>
   <zorder>but_next</zorder>
   <zorder>but_file</zorder>
   <zorder>label_text</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
